src/services/codefixes/importFixes.ts

```diff
@@ -241,7 +241,7 @@ export function createImportAdder(sourceFile: SourceFile | FutureSourceFile, pro
 interface AddToExistingState {
     readonly importClauseOrBindingPattern: ImportClause | ObjectBindingPattern;
     defaultImport: Import | undefined;
-    readonly namedImports: Map<string, AddAsTypeOnly>;
+    readonly namedImports: Map<string, [AddAsTypeOnly, /*propertyName*/ string | undefined]>;

```

Comment:
Nit: tuples support real labels! Although, maybe an object type would be better at that point.